examples/edge_intelligence/windmill_tf_edge.py

```diff
@@ -0,0 +1,144 @@
+#  - Redistributions of source code must retain the above copyright notice,   #
+#      this list of conditions and the following disclaimer.                  #
+#                                                                             #
+#  - Redistributions in binary form must reproduce the above copyright        #
+#      notice, this list of conditions and the following disclaimer in the    #
+#      documentation and/or other materials provided with the distribution.   #
+#                                                                             #
+#  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"#
+#  AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE  #
+#  IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE #
+#  ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE  #
+#  LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR        #
+#  CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF       #
+#  SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS   #
+#  INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN    #
+#  CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)    #
+#  ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF     #
+#  THE POSSIBILITY OF SUCH DAMAGE.                                            #
+# ----------------------------------------------------------------------------#
+
+# This example depicts how edge component can be used to take actions locally based on trained TensorFlow model
+# and actions can then be send to device using actuator_udm method defined here, which is currently just printing
+# actions on console, metrics are sent to Graphite DCC
+
+import math
+import random
+
+from linux_metrics import cpu_stat, mem_stat
+from liota.dccs.graphite import Graphite
+from liota.entities.metrics.metric import Metric
+from liota.entities.edge_systems.dell5k_edge_system import Dell5KEdgeSystem
+from liota.dcc_comms.socket_comms import SocketDccComms
+from liota.dccs.dcc import RegistrationFailure
+from liota.edge_component.tensorflow_edge_component import TensorFlowEdgeComponent 
+from liota.lib.utilities.utility import read_user_config
+
+# getting values from conf file
+config = {}
+execfile('../sampleProp.conf', config)
+
+def read_cpu_procs():
+	return cpu_stat.procs_running()
+
+def read_cpu_utilization(sample_duration_sec=1):
+	cpu_pcts = cpu_stat.cpu_percents(sample_duration_sec)
+	return round((100 - cpu_pcts['idle']), 2)
+
+def read_mem_free():
+	total_mem = round(mem_stat.mem_stats()[1], 4)
+	free_mem = round(mem_stat.mem_stats()[3], 4)
+	mem_free_percent = ((total_mem - free_mem) / total_mem) * 100
+	return round(mem_free_percent, 2)
+
+# actuator_udm can be used to pass on the value to the actuator, as of now we are printing them
+def actuator_udm(value):
+	print value
+
```

Comment:
Simulating windmill RPM which is a device connected to Edge.

Reply:
Done.